src/engraving/draw/painter.h

```diff
@@ -198,9 +199,9 @@ class Painter
     std::stack<State> m_states;
 };
 
-inline void Painter::setPen(const QColor& color)
+inline void Painter::setPen(const mu::draw::Color& color)
```

Comment:
There is no need to specify the namespace mu::draw::, they are in the same namespace